spyder/plugins/appearance/confpage.py

```diff
@@ -56,7 +56,7 @@ def setup_page(self):
         )
         self.ui_combobox = ui_theme_combo.combobox
 
-        themes = ['Spyder 2', 'Spyder 3']
+        themes = ['Spyder 3']
```

Comment:
Since it won't be possible to change the icon theme anymore, I suggest to remove this combobox from Preferences.

We should also remove the associated option (i.e. `icon_theme` from our config system).